cli-files/go-cat/cmd/root_test.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestRootCmd(t *testing.T) {
+	t.Run("No path provided", func(t *testing.T) {
+
+		cmd := rootCmd
+		err := cmd.Execute()
+
+		if err == nil {
+			t.Error("Expected error, got nil")
+		}
+	})
+
+	t.Run("path to file dew.txt", func(t *testing.T) {
+		r, w, _ := os.Pipe()
```

Comment:
Even in tests, ignoring errors isn't great practice - it can obfuscate the cause of an error (let's say this call failed - you may get a confusing error later on about not being able to write to a pipe, whereas if you'd failed the test early because the pipe couldn't be created, it's very clear what's wrong)